code/modules/mob/living/carbon/human/human.dm

```diff
@@ -851,6 +851,14 @@
 	update_icons()	//apply the now updated overlays to the mob
 
 
+/mob/living/carbon/human/wash_cream()
+	var/image/creamoverlay = image('icons/effects/creampie.dmi')
+	var/mob/living/carbon/human/H = src
+	if(H.dna && H.dna.species && ("tail_lizard" in H.dna.species.mutant_bodyparts))
+		creamoverlay.icon_state = "creampie_lizard"

```

Comment:
why do you absolutely want to check for the lizard's tail? DOes the overlay cover the tail? If not, you can just check H.dna.species.id == "lizard" , it's faster.